tests/unit/workflows/test_workflows_tasks_actions.py

```diff
@@ -651,9 +649,10 @@ def test_refextract_from_pdf(mock_get_pdf_in_workflow):
     assert obj.data['references'][0]['raw_refs'][0]['source'] == 'arXiv'
 
 
-@patch('inspirehep.modules.workflows.tasks.actions.get_pdf_in_workflow')
-def test_refextract_from_text(mock_get_pdf_in_workflow):
-    mock_get_pdf_in_workflow.return_value = None
+@patch('inspirehep.modules.workflows.tasks.actions.get_document_in_workflow')
+def test_refextract_from_text(mock_get_document_in_workflow):
+    mock_get_document_in_workflow.return_value.__enter__.return_value = None
+    mock_get_document_in_workflow.return_value.__exit__.return_value = None
```

Comment:
is it needed to change the `__exit__` return value? Wouldn't it be just discarded?

Reply:
`__exit__` is ran at the end of the context manager, and before I added this line I was getting errors because of it (don't remember the exact error though).

Reply:
👍